config/locales/en.yml

```diff
@@ -74,6 +74,23 @@ en:
       sign_in: Sign In
       sign_out: Sign Out
   patient:
+    abortion:
```

Comment:
thoughts about changing this to `abortion_information` so it matches the partial? or is that overkill?